CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#806](https://github.com/nf-core/sarek/pull/806) - Refactor all tests
 - [#806](https://github.com/nf-core/sarek/pull/806) - Split up `modules.config` file
 - [#810](https://github.com/nf-core/sarek/pull/810) - Update CHANGELOG
```

Comment:
What's happening here?

Reply:
🤨 was that you test run?

Reply:
https://github.com/nf-core/sarek/pull/810 I think it was, can prob remove that line

Reply:
Did we merge it?

Reply:
yes it was merged in 
![Screenshot 2022-11-10 at 00 21 49](https://user-images.githubusercontent.com/12273093/200963225-e6735fa2-c8b1-41d4-b07b-4adff5db188e.png)